client/src/locales/en-US.json

```
@@ -149,7 +149,12 @@
                 "hours": "Less than one hour | {hours} hours remaining",
                 "expired": "Expired"
             },
-            "noLastLogin": "--"
+            "noLastLogin": "--",
+            "errors": {
+                "noStorage": "We cannot access the application storage. We are not able to list or save your organizations.",
```

I would start the messages by the problem the user is facing ("Unable to...") and then the technical reason ("storage not available).

```suggestion
                "noStorage": "Unable to access your organizations: the application storage is not available.",
```